### PR TITLE
FUSETOOLS-2804: UI Tests - Create a new 'quick' test for component 'Properties view'

### PR DESCRIPTION
**Pull request by djelinek**
# Pull Request Checklist

After this checklist is all checked or PR provides explanations for possible pass-through, please put the label "Ready for review" on the PR.

## General

- [ ] Did you use the Jira Issue number in the commit comments?
- [ ] Did you set meaningful commit comments on each commit?
- [ ] Did you sign off all commits for this PR? (git commit -s -m "jira issue number - your commit comment")

## Functional

- [ ] Did the CI job report a successful build?
- [ ] Is the non-happy path working, too?

## Maintainability

- [ ] Are all Sonar reported issues fixed or can they be ignored?
- [ ] Is there no duplicated code?
- [ ] Are method-/class-/variable-names meaningful?

## Tests

- [ ] Are there unit-tests?
- [ ] Are there integration tests (or at least a jira to tackle these)?
- [ ] Do we need a new UI test?

## Legal

- [ ] Have you used the correct file header copyright comment?
- [ ] Have you used the correct year in the headers of new files?

### CODE REVIEW

**Comment by djelinek** [comment]
please do not merge until [FUSETOOLS-2806](https://github.com/jbosstools/jbosstools-fuse/pull/1335) is merged

**Comment by apupier** [comment]
testPR

**Comment by tsedmik** [comment]
testPR

File: uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/CamelCatalogUtils.java [hunk 1]
```diff
@@ -26,13 +33,30 @@
...
+
+	private static String LANGUAGES_PATH = ROOT_PATH + "languages";
+
+	private static String LANGUAGES_LIST_PATH = ROOT_PATH + "languages.properties";
+
+	private static String MODELS_PATH = ROOT_PATH + "models";
+
+	private static String MODELS_LIST_PATH = ROOT_PATH + "models.properties";
+
+	private static final String JSON_SUFFIX = ".json";
```
Comment (tsedmik) [inline uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/CamelCatalogUtils.java]: All constants should be public. There is no need to hide them.

File: uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/CamelCatalogUtils.java [hunk 2]
```diff
@@ -84,6 +114,33 @@ public String Components() {
 		return getCatalogComponentsList(CatalogType.COMPONENT);
 	}
 
+	/**
+	 * Returns all available dataformats
+	 *
+	 * @return String
+	 */
+	public String DataFormats() {
```
Comment (tsedmik) [inline uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/CamelCatalogUtils.java]: please follow Java convention
> Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

File: uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/CamelCatalogUtils.java [hunk 3]
```diff
@@ -84,6 +114,33 @@ public String Components() {
...
+		return getCatalogComponentsList(CatalogType.DATAFORMAT);
+	}
+
+	/**
+	 * Returns all available languages
+	 *
+	 * @return String
+	 */
+	public String Languages() {
```
Comment (tsedmik) [inline uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/CamelCatalogUtils.java]: please follow Java conventions

File: uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/CamelCatalogUtils.java [hunk 4]
```diff
@@ -84,6 +114,33 @@ public String Components() {
...
+		return getCatalogComponentsList(CatalogType.LANGUAGE);
+	}
+
+	/**
+	 * Returns all available models
+	 *
+	 * @return String
+	 */
+	public String Models() {
```
Comment (tsedmik) [inline uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/CamelCatalogUtils.java]: please follow Java conventions

File: uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/view/FusePropertiesView.java [hunk 5]
```diff
@@ -185,27 +178,47 @@ else if (items.contains(value))
...
 
-	public Map<Widget, Widget> getDetailsProperties() {
+	public List<String> getPropertiesLabelsList(final String tab) {
+		List<String> properties = new ArrayList<>();
+		Map<Widget, Widget> widgets = getAdvancedProperties(tab);
+		for (Widget w : widgets.keySet()) {
+			if(w instanceof Label)
+				properties.add(getLabel(w));
+		}
+		/*
```
Comment (tsedmik) [inline uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/view/FusePropertiesView.java]: do not leave unused Java code in code base - please remove commented-out Java code

File: uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/CamelCatalogComponentsTest.java [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+ */
+@CamelCatalog
+@CleanWorkspace
+@CleanErrorLog
+@RunWith(RedDeerSuite.class)
+@OpenPerspective(FuseIntegrationPerspective.class)
+@UseParametersRunnerFactory(ParameterizedRequirementsRunnerFactory.class)
+public class CamelCatalogComponentsTest {
+
+	private static final String PROJECT_NAME = "cbr";
```
Comment (tsedmik) [inline uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/CamelCatalogComponentsTest.java]: make all constants public available

File: uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/CamelCatalogUtils.java [hunk 7]
```diff
@@ -109,7 +110,7 @@ public CamelCatalogUtils(String path) {
 	 * 
 	 * @return String
 	 */
-	public String Components() {
+	public List<String> components() {
```
Comment (tsedmik) [inline uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/CamelCatalogUtils.java]: Please, use verbs in names of methods

**Pull request merged**